pkg/model_registry/service/registered_models.go

```diff
@@ -0,0 +1,175 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/mlflow/mlflow-go/pkg/contract"
+	"github.com/mlflow/mlflow-go/pkg/entities"
+	"github.com/mlflow/mlflow-go/pkg/protos"
+)
+
+var (
+	RegisteredModelAliasRegex        = regexp.MustCompile(`^[\w\-]*$`)
+	RegisteredModelAliasVersionRegex = regexp.MustCompile(`^[vV]\d+$`)
+)
+
+func (m *ModelRegistryService) UpdateRegisteredModel(
+	ctx context.Context, input *protos.UpdateRegisteredModel,
+) (*protos.UpdateRegisteredModel_Response, *contract.Error) {
+	registeredModel, err := m.store.UpdateRegisteredModel(ctx, input.GetName(), input.GetDescription())
+	if err != nil {
+		return nil, err
+	}
+
+	return &protos.UpdateRegisteredModel_Response{
+		RegisteredModel: registeredModel.ToProto(),
+	}, nil
+}
+
+func (m *ModelRegistryService) RenameRegisteredModel(
+	ctx context.Context, input *protos.RenameRegisteredModel,
+) (*protos.RenameRegisteredModel_Response, *contract.Error) {
+	newName := input.GetNewName()
+	if newName == "" {
```

Comment:
Nitpick, but could you check these as well or create an issue to follow-up on this?